app/client/components/user/CreateUser.tsx

```diff
@@ -30,16 +32,19 @@ interface OuterProps {
   isVendor: boolean;
 }
 
-declare global {
-  interface Window {
-    BENEFICIARY_TERM: string;
```

Comment:
we should really audit how we are handling this across the software - it's not consistent

Reply:
Agreed that this needs some thought - assuming that setting the term for a beneficiary is distinct from doing translations, what other handling methods do we have? and where?

Reply:
Let's resolve this with https://trello.com/c/bGS17Tyz/511-consistent-and-explicit-labelling-of-user-types-across-entire-platform